content/applications/finance/fiscal_localizations/argentina.rst

```diff
@@ -948,6 +948,87 @@ the corresponding layout and fiscal stamps stated in the :ref:`Invoice printed r
 .. seealso::
    :doc:`Client account creation <../../websites/ecommerce/checkout_payment_shipping/checkout>`
 
+Liquidity product direct sales
+------------------------------
+
+The liquidity product direct sale is used in cases where sales are made to a third party. In these
+cases, the seller and the proprietary company of the goods can register their corresponding sales
+and purchase. This feature is included when :ref:`installing <general/install>` the *Argentinian
+Electronic Invoicing* module (`l10n_ar_edi`).
+
+Configuration
+~~~~~~~~~~~~~
+
+Document types
+**************
+
+This feature includes the addition of the following document types: 23, 24, 25, 26, 27, 28, 33, 43,
+45, 46, 48, 58, 60, 61, 150, 151, 157, 158, 161, 162, 164, 166, 167, 171, 172, 180, 182, 186, 188,
+332.
+
+.. important::
+   Note that by default these document types are *inactive*. For the workflow to work, it is
+   necessary to *activate* these documents.
+
+To activate them, navigate to :menuselection:`Accounting --> Configuration --> Document Types` and
+remove any active filters. Then, find the documents and activate them by toggling the option
+:guilabel:`Active`.
```

Comment:
The toggle don't work for me (17.0 runbot, 17.1 test db, base with the required modules installed, but not the full configuration)

Reply:
@xpl-odoo @samueljlieber you are right, in that case, I suggest the following:

Then, find the documents and activate them by selecting them (checkbox), go to :guilabel:'Actions > Unarchive'.